packages/management/tkg/bundle/config/packageinstalls/service-account.yaml

```diff
@@ -5,14 +5,14 @@ metadata:
   name: tanzu-tkg-package-sa
   namespace: tkg-system
   annotations:
-    kapp.k14s.io/change-group: "v1/sa-tkg-pkg"
+    kapp.k14s.io/change-group: "sa-tanzu-tkg-package"
```

Comment:
since this SA is being used to install `tkg-clusterclass` package, can we change the value of this annotation to follow this convention ``packageName-packageinstall/serviceaccount` ie`tkg-clusterclass-packageinstall/serviceaccount` to be more clear? I tried to do the same for tanzu-framework-manager meta package https://github.com/vmware-tanzu/tanzu-framework/pull/1848/files#diff-b0f4231cf94dd415b922ab3391584ef24dcbe3ebd251907d4853c07bad2a9a6fR8

I'm open to other suggestions too, but just want to create a convention so that other package authors can follow the same pattern to avoid confusion later..